server/src/Config.hs

```diff
@@ -59,6 +60,12 @@ configParser environment = Config
                   metavar "HOST:PORT" <>
                   help "If provided, Icepeak collects various metrics and provides them to Prometheus on the given endpoint."
                  ))
+  <*> option auto
+       (long "queue-capacity" <>
+        metavar "INTEGER" <>
+        value 256 <>
+        help ("Higher values allow more requests to come in within a short amount of time. " <>
+              "Smaller values increase the risk of data loss."))
```

Comment:
s/increase/decrease

If the queue is smaller, fewer values can be lost.

Reply:
Right, but you also lose more requests. Will clarify and merge.